docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -98,7 +104,7 @@ Once you have chosen your scenario, Model Builder asks you to provide a dataset.
 
 ![Diagram showing Model Builder steps](media/model-builder-steps.png)
 
-Model Builder supports datasets in .tsv, .csv, .txt formats, as well as SQL database format. If you have a .txt file, columns should be separated with `,`, `;` or `/t` and the file must have a header row.
+Model Builder supports datasets in .tsv, .csv, .txt formats, as well as SQL database format. If you have a .txt file, columns should be separated with `,`, `;` or `/t`.
```

Comment:
Is the delimiter correct? I think it's `\`

```suggestion
Model Builder supports datasets in .tsv, .csv, .txt formats, as well as SQL database format. If you have a .txt file, columns should be separated with `,`, `;` or `\t`.
```

Reply:
Good catch!